### ISSUE TITLE
Support for Ignition?

### ISSUE THREAD

**Issue by SteveMacenski** [aws-robotics/aws-robomaker-small-warehouse-world#22]
Starting in 22.04 (ROS2 Humble; new LTS distribution being released on Monday) Gazebo no longer exists in the ROS ecosystem, its now only Ignition. As part of the migration from Gazebo to Ignition for Nav2, we'd like to use this as our default bringup. We have alot of work invested already to update all our tests and default bringup to use this work in Gazebo in a pending PR, but for the Humble migration it would need to be in Ignition.

I don't think its a huge amount of work, I believe all the assets should render OK, just minor updates to the world files and new ignition launch files

**Comment by SteveMacenski** [comment]
Any update? I don't think its too much effort, just needs someone to do it.

**Comment by gy2256** [comment]
I was trying to open the small_warehouse.world following the documentation and end up with an empty world. 
`export GAZEBO_MODEL_PATH=pwd/models`
`gazebo small_warehouse.world`

My OS is Ubuntu 22.04 with ROS2 Humble, is this the reason that assets are not loading?

**Comment by Minipada** [comment]
Using Galactic and having the repository in my workspace I had to

```bash
source /opt/ros/galactic/setup.bash
source install/setup.bash 
source /usr/share/gazebo/setup.bash
export GAZEBO_RESOURCE_PATH=${GAZEBO_RESOURCE_PATH}:${PWD}/src/aws_robomaker_small_warehouse_world
# Then
gazebo src/aws_robomaker_small_warehouse_world/worlds/small_warehouse/small_warehouse.world --verbose
```

I don't know how to fix it *permanently*

**Comment by borongyuan** [comment]
I have migrated AWS warehouse world to Ignition Gazebo. I think it's better to create a new branch instead of merging directly into the ros2 branch.
https://github.com/borongyuan/aws-robomaker-small-warehouse-world

**Comment by SteveMacenski** [comment]
Agreed, someone with maintainer rights needs to open a new branch for you to open a PR!

**Comment by Mechazo11** [comment]
Hi @borongyuan great work in porting this project to Ignition. Do you have any plans for porting it to newer Gazebo i.e. Gazebo Harmonic?

Best,
@Mechazo11

**Comment by izzaaaatanishq** [comment]
@Mechazo11 i have updated it 
PR: https://github.com/aws-robotics/aws-robomaker-small-warehouse-world/pull/28
